google/resource_kms_crypto_key.go

```diff
@@ -42,6 +43,16 @@ func resourceKmsCryptoKey() *schema.Resource {
 	}
 }
 
+func kmsCryptoKeyRingsEquivalent(k, old, new string, d *schema.ResourceData) bool {
```

Comment:
Hum, I thought the whole point of #1322 was to have a consistent id.

Reply:
And we still do, but the user might provide the long form in their config.  See how this only tries to parse the long-form out of `new` (e.g. the value in the config) instead of `old` (e.g. the value in state).